lib/raven/configuration.rb

```diff
@@ -198,6 +202,7 @@ def initialize
       self.project_root = detect_project_root
       self.rails_activesupport_breadcrumbs = false
       self.rails_report_rescued_exceptions = true
+      self.retryable_exceptions = []
```

Comment:
These changes fail two lint rules:
1. `Assignment Branch Condition size for initialize is too high`, can we just increase this from 30 to ~32? Otherwise, I'm not sure if there's a better place to put this configuration.
2. `Class has too many lines`, not sure if you're OK increasing this limit as well, or if we should try pulling out some of the common functionality. For example, `retryable_exception?` and `excluded_exception?` share logic using `get_exception_class`, which we could pull out into a `util` or similar.